utils/wasm-builder/src/lib.rs

```diff
@@ -102,17 +149,42 @@ impl Default for WasmBuilder {
     }
 }
 
+// The `std` feature is excluded by default because it is usually used for building
+// so called WASM wrapper which is a static library exposing the built WASM.
+const FEATURES_TO_EXCLUDE_BY_DEFAULT: &[&str] = &["std"];
+
 /// Shorthand function to be used in `build.rs`.
 pub fn build() {
-    WasmBuilder::new().build();
+    build_custom(FEATURES_TO_EXCLUDE_BY_DEFAULT)
+}
+
+/// Shorthand function similar to the [`build`] one, but allowing some customizations.
+pub fn build_custom(features_to_exclude: &[&'static str]) {
```

Comment:
Too common function name.
```suggestion
pub fn build_with_scoped_features(scoped_features: &[&'static str]) {
```

Reply:
The idea behind this name is making it as a fully customisable shortcut, i.e. if anything else needs to be added as a customisation, it will be added as yet another param of this function. It eliminates the need of making up some weird function names. You won't be able to come up with a set of shortcuts which cover all possible use cases.

Reply:
Any addition to this function will break backward compatibility. Is it a sense to make such kind of function?

Reply:
Nice catch!!! I will scrap it away then. Let people use underlaying builder directly in case of customisations.